scripts/setonix.profile

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
-source /opt/cray/pe/cpe/22.09/restore_lmod_system_defaults.sh
+source /opt/cray/pe/cpe/23.03/restore_lmod_system_defaults.sh
```

Comment:
I just re installed quokka on setonix. When I now do `source scripts/setonix.profile`, i get:

```
psharda@setonix-01:/software/projects/pawsey0807/psharda/quokka> source scripts/setonix.profile
-bash: /opt/cray/pe/cpe/22.09/restore_lmod_system_defaults.sh: No such file or directory
Lmod has detected the following error:  The following module(s) are unknown: "cmake/3.21.4"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "cmake/3.21.4"

Also make sure that all modulefiles written in TCL start with the string #%Module



Lmod has detected the following error:  The following module(s) are unknown: "rocm/5.0.2"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "rocm/5.0.2"

Also make sure that all modulefiles written in TCL start with the string #%Module



Lmod has detected the following error:  The following module(s) are unknown: "cce/14.0.3"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "cce/14.0.3"

Also make sure that all modulefiles written in TCL start with the string #%Module

```

Reply:
That's the old `setonix.profile`.

Reply:
oops